draft-ietf-quic-applicability.md

```diff
@@ -466,6 +466,9 @@ based on the Connection ID (see also {{connid}}) in the QUIC header,
 if present. This supports cases where address information changes, such as
 NAT rebinding, intentional change of the local interface, or based on an
 indication in the handshake of the server for a preferred address to be used.
+As such if the client knows or is likely to sit behind a NAT, use of the
```

Comment:
Even if the client knows, it can't communicate that to the server in v1.

```suggestion
As such if the client is known or likely to sit behind a NAT, use of the
```